frontend/testing/cypress/src/integration/studio/designer.js

```diff
@@ -43,24 +43,48 @@ context('Designer', () => {
 
     // Add new page and ensure updated data is loaded
     designer.getAddPageButton().click();
+
     cy.wait('@postLayoutSettings').its('response.statusCode').should('eq', 200);
     cy.wait('@getLayoutSettings').its('response.statusCode').should('eq', 200);
 
-    // Verify navigation button exists in form
-    designer.getDroppableList().findByRole('listitem', { name: texts['ux_editor.component_navigation_buttons'] }).should('be.visible');
-
     // Add an input component
     designer.getToolbarItemByText(texts['ux_editor.component_input']).trigger('dragstart');
     designer.getDroppableList().trigger('drop');
     cy.wait(500);
-    designer.getDroppableList()
+    designer
+      .getPageAccordionByName('Side1')
       .findAllByRole('listitem')
-      .then(($elements) => expect($elements.length).eq(2));
+      .then(($elements) => expect($elements.length).eq(1));
 
     // Delete components on page
     cy.deleteComponents();
   });
 
+  it('should add navigation buttons when adding more than one page', () => {
+    cy.intercept('GET', '**/app-development/layout-settings?**').as('getLayoutSettings');
+    cy.intercept('POST', '**/app-development/layout-settings?**').as('postLayoutSettings');
+
+    // Navigate to designerApp
+    cy.visit('/editor/' + designerAppId);
+    header.getCreateLink().click();
+    cy.ensureCreatePageIsLoaded();
+
+    // Add two new pages to ensure that navigation-buttons will be added to page
+    designer.getAddPageButton().click();
+    designer.getAddPageButton().click();
+
+    cy.wait('@postLayoutSettings').its('response.statusCode').should('eq', 200);
+    cy.wait('@getLayoutSettings').its('response.statusCode').should('eq', 200);
+
+    // Add an input component
+    cy.wait(500);
+    designer
+      .getPageAccordionByName('Side2')
+      .findByRole('listitem', { name: `${texts['ux_editor.component_navigation_buttons']}` });
+
+    cy.deleteComponents();
```

Comment:
Testet dette lokalt hos meg og det fungerer fint. Var det her du fikk problemer? 🤔 
```suggestion
  designer
      .getPageAccordionByName('Side1')
      .findByRole('listitem', { name: texts['ux_editor.component_input'] });
```